### PR TITLE
fix: loading tips german grammar

### PR DESCRIPTION
**Pull request by nyhmbo**
## Summary of Changes
small grammar fix for the german loading tips translation

## Screenshots (if necessary)
-

## References
-

## Additional context

Discord username (if different from GitHub): Nyhmbo#9999

## Testing instructions
-

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by Benjozork — APPROVED** [review]
LGTM

**Comment by Saschl** [comment]
Correction approved ;)

**Review by FoxtrotSierra6829 — APPROVED** [review]

File: A32NX/de-DE.locPak [hunk 1]
```diff
@@ -14,7 +14,7 @@
       "A32NX.LOADING.TIPS.A320NEO_003": "FlyByWire ist nach der Fly-by-Wire-Steuerung benannt, die erstmals im Airbus A320 eingeführt wurde.",
       "A32NX.LOADING.TIPS.A320NEO_004": "Es sind 9693 Flugzeuge der A320-Familie gebaut worden! Das ist fast so viel wie die Bevölkerung von Tuvalu!",
       "A32NX.LOADING.TIPS.A320NEO_005": "Airbus A320neo Flugzeuge haben eine Reichweite von 3500 NM oder 6500 KM. Das ist genug, um von New York nach Paris zu fliegen!",
-      "A32NX.LOADING.TIPS.A320NEO_006": "Wussten Sie schon? Der A320-Familie hat 4 Varianten! Der A318 A319 A320 und A321."
+      "A32NX.LOADING.TIPS.A320NEO_006": "Wussten Sie schon? Die A320-Familie hat 4 Varianten! Den A318 A319 A320 und A321."
```
Comment (NathanInnes) [inline A32NX/de-DE.locPak:17]:
```suggestion
      "A32NX.LOADING.TIPS.A320NEO_006": "Wussten Sie schon? Die A320-Familie hat 4 Varianten! Den A318, A319, A320, und A321."
```

Not sure if this applies in German but wouldn't you comma separate these items?

Comment (nyhmbo) [reply A32NX/de-DE.locPak]: good suggestion. makes sense ofc

**Review by MisterChocker — APPROVED** [review]
LGTM

**Comment by MisterChocker** [comment]
Small enough, merging now.

**Comment by MisterChocker** [comment]
Small enough change, merging directly.

**Pull request merged**